draft-ietf-rtcweb-jsep.xml

```diff
@@ -607,7 +607,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             used instead.</t>
 
             <t>When creating an IceCandidate object, JSEP
-            implementations MUST populate all of these fields.</t>
+            implementations MUST populate the aforementioned
```

Comment:
actually, I think this may require us to also say, "though MUST also be prepared to receive candidates with some of the fields missing, as mentioned above".